integrations/helm/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- bencer
+- nestorsalceda
```

Comment:
It seems @nestorsalceda can't dedicate effort to this atm (sadly) ([comment here](https://github.com/falcosecurity/contrib/issues/12#issuecomment-627410685)).

So we should remove him from here, but I strongly think he should be left as a **reviewer** since he's the author of the Helm.

Reply:
Also, in this file we should add other people that expressed the will to maintain and evolve it. Ie., @nibalizer